websites/N/Necord/presence.ts

```diff
@@ -0,0 +1,131 @@
+enum SmallImageKeys {
+	Idle = "https://i.imgur.com/6A9pxAR.png",
+	Search = "https://i.imgur.com/T9NzckB.png",
+	Read = "https://i.imgur.com/ULFqucm.png",
+}
+
+const presence = new Presence({
+		clientId: "1081479845940314114",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000),
+	IDLE_TIMEOUT = 10 * 60 * 1000,
+	settings = [
+		"search",
+		"docs-page",
+		"docs-page-content",
+		"docs-page-sidebar",
+		"idling",
+	];
+
+let sidebar: string = null,
+	lastActivity: number = Date.now();
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			largeImageKey: "https://i.imgur.com/LY804vK.png",
+			startTimestamp: browsingTimestamp,
+		},
+		{ href, pathname, search } = document.location,
+		title = document.title.split(" | ")[0],
+		[
+			isSearchPublic,
+			isCurrentPagePublic,
+			isCurrentContentPublic,
+			isSidebarPublic,
+			isIdlingPublic,
+		] = await Promise.all(
+			settings.map(setting => presence.getSetting(setting))
+		);
+
+	switch (true) {
+		case lastActivity + IDLE_TIMEOUT < Date.now() && isIdlingPublic: {
+			presenceData.smallImageKey = SmallImageKeys.Idle;
+			presenceData.smallImageText = "Idling...";
```

Comment:
Remove `...` from strings
```suggestion
			presenceData.smallImageText = "Idling";
```

Reply:
Note that this also applies to the other cases of this.

Reply:
Applied at #[ee33ea4](https://github.com/PreMiD/Presences/pull/7215/commits/ee33ea48b78bb03ff93f780ddb1769b474ae7d70)